rate_limit/backend.py

```diff
@@ -28,9 +28,10 @@
 class Backend(object):
     """Backend for storing rate limits."""
 
-    def __init__(self, host, port, rate_limit_response, logger=log.Logger(__name__), **kwargs):
+    def __init__(self, host, port, password, rate_limit_response, logger=log.Logger(__name__), **kwargs):
         self.__host = host
         self.__port = port
+        self.__password = password
         self.__rate_limit_response = rate_limit_response
         self.logger = logger
```

Comment:
We're setting this in RedisBackend, so this gets overwritten. But it also looks like the `Backend` class doesn't do anything at all. We could maybe convert it to a abstract baseclass or remove it once for all.

For the sake of consistency we should probably keep it as is, just something to keep in mind for the next refactor.